src/components/nodes/pages/stanford-page/stanford-page-page.tsx

```diff
@@ -30,7 +31,7 @@ const StanfordPagePage = ({node, ...props}: Props) => {
       )}
 
       {node.suPageBanner?.__typename !== "ParagraphStanfordPageTitleBanner" && (
-        <H1 className="centered mt-32">{node.title}</H1>
+        <H1 className={clsx("centered mt-32", {"lg:max-w-1200": fullWidth})}>{node.title}</H1>
```

Comment:
But what about non-full width pages that aren't in the menu?

Reply:
I check in with Kerri on this! She will inform Chris and document how content authors can achieve the correct Page title H1 alignment 😄